CMakeLists.txt

```diff
@@ -96,6 +97,10 @@ endif()
 add_library(tg_owt)
 init_target(tg_owt)
 
+if (TG_OWT_USE_X11)
```

Comment:
Why this is moved?

Reply:
If there's no real reason, please just return it to the original position

Reply:
Sorry for the late reply, it's been a while, but it was either to chain better with some other patches I was applying at the time, or because the order actually mattered, I don't remember which.